src/main/webapp/src/app/common/growpod-page-styles.css

```diff
@@ -9,3 +9,21 @@
 .text-center {
   text-align: center;
 }
+
+.profile-pic-container-s {
+  position: relative;
+  width: 33%; /* The size you want */
+  margin-left: auto;
```

Comment:
`margin-left` and `margin-right` can be consolidated if they are the same value

Reply:
Will change to `margin: 0px auto`.

Reply:
nit: `0px;` doesn't actually need the unit. can write `0;`

Reply:
Done.